bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/handler/PulseaudioHandler.java

```diff
@@ -104,7 +99,14 @@ public void initialize() {
         name = (String) config.get(DEVICE_PARAMETER_NAME);
 
         updateStatus(ThingStatus.UNKNOWN);
-        deviceOnlineWatchdog();
+        PulseaudioBridgeHandler pulseaudioBridgeHandler = getPulseaudioBridgeHandler();
+        if (pulseaudioBridgeHandler != null) {
+            pulseaudioBridgeHandler.waitForInitialization();
```

Comment:
This is not good as initialize() must return fast. You cannot lock this thread waiting for another thing  handler to be initialized.